vet.sh

```diff
@@ -88,6 +88,9 @@ git grep -l '"math/rand"' -- "*.go" 2>&1 | not grep -v '^examples\|^stress\|grpc
 # - Ensure all ptypes proto packages are renamed when importing.
 not git grep "\(import \|^\s*\)\"github.com/golang/protobuf/ptypes/" -- "*.go"
 
+# - Ensure all xds proto imports are renamed to *pb or *grpc.
+git grep '"github.com/envoyproxy/go-control-plane/envoy' -- '*.go' | not grep -v 'pb "\|grpc "'
```

Comment:
Is there a missing double-quote around `github.com/envoyproxy/go-control-plane/envoy`?

Reply:
This is the prefix. The whole string would something after `envoy`, like
```
	corepb "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
```

Reply:
Makes sense.